apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -110,18 +110,20 @@ NetSimLocalClientNode.inherits(NetSimClientNode);
 /**
  * Static async creation method. See NetSimEntity.create().
  * @param {!NetSimShard} shard
+ * @param {string} displayName
  * @param {!NodeStyleCallback} onComplete - Method that will be given the
  *        created entity, or null if entity creation failed.
  */
-NetSimLocalClientNode.create = function (shard, onComplete) {
-  NetSimEntity.create(NetSimLocalClientNode, shard, function (err, node) {
+NetSimLocalClientNode.create = function (shard, displayName, onComplete) {
+  var templateNode = new NetSimLocalClientNode(shard);
+  templateNode.displayName_ = displayName;
+  templateNode.getTable_().create(templateNode.buildRow_(), function (err, row) {
```

Comment:
I think we've maybe had this discussion before :)

I can understand the justification for accessing "privates" here. However, as soon as we allow this, it becomes more difficult for my internal linter to know when it is/isn't acceptable, as I now need to have an understanding of the class hierarchy to make a decision about whether or not this is a good usage. For that reason, I think my preference would be to name these as publics, or come up with some alternative indicator for "protected" (which is probably overkill)